forecast-admin/forecast/opportunities/fixtures/create_offices.py

```diff
@@ -0,0 +1,49 @@
+import json
```

Comment:
I'd put a docstring at the very beginning of this script indicating when this script should be used, and how.